client/modules/workspace/src/lib/workspace.tsx

```diff
@@ -12,11 +13,18 @@ export function Workspace(props: WorkspaceProps) {
       <h1>Welcome to Workspace!</h1>
       {isLoading && <div>Loading app listing...</div>}
       {data && (
-        <ul>
-          {data.map((appMeta) => (
-            <li key={appMeta.uuid}>{appMeta.value.definition.id}</li>
+        <>
+          {data.categories.map((category) => (
+            <div key={uuidv4()}>
```

Comment:
Random UUIDs as keys should be a last resort. There should be a way to uniquely identify a category/app based on the API response.

Reply:
For sure, this was for speed. I resolved this in the new branch